test/rules/avoid_redundant_argument_values.dart

```diff
@@ -56,4 +58,10 @@ void main() {
   gg(1, 0); //LINT
   gg(1, 1); //OK
   gg(1); //OK
+
+  ggg(
+      1, // OK the  - first argument is required so that we can provide the second argument.
+      3);
+  ggg(1,  // LINT
```

Comment:
The code motion change above will cause this to not be a lint, but I think that's better.

Reply:
Agreed.